Added visanju mevans HNeff shoobe01 A11yReadTech

Myndex opened the pull request.

Myndex committed:
Added visanju mevans HNeff shoobe01 A11yReadTech

joe-watkins commented:
@Myndex Thanks for the PR. After reviewing the folks you've added to the list I'm gonna pass on the addition of this particular person's Twitter profile: Steven Hoober [@shoobe01](https://twitter.com/shoobe01) Too many Fox News / Political Retweets. Even if I personally may or may not align with them, the content isn't centered enough around accessibility, digital, web, design, etc. Perhaps this person has a professional account we could reference?

Happy to take the PR without that account addition though :)

Myndex committed:
Remove a name

Myndex commented:
> _Thanks for the PR. After reviewing the folks you've added to the list I'm gonna pass on the addition of this particular person's Twitter profile: Steven Hoober [@shoobe01](https://twitter.com/shoobe01) Too many Fox News / Political Retweets._ 

Hi @joe-watkins 

Regarding Steve, I added him because of his book http://www.4ourth.com/writing/ ... I didn't realize his account was not as much about accessibility/design, as that is what I primarily interact with him about, and I was more thinking about his book. I updated the PR as requested, please let me know if there's anything else?

Thank you 😌

joe-watkins commented:
tnx @Myndex I hate to have to sensor things,.. I just do a quick scan of things before merging and that particular account didn't match the quality/vibe of other great additions you had. Thank you for understanding :)
